src/recursive_sorting/recursive_sorting.py

```diff
@@ -1,17 +1,31 @@
-# TO-DO: complete the helpe function below to merge 2 sorted arrays
-def merge( arrA, arrB ):
-    elements = len( arrA ) + len( arrB )
-    merged_arr = [0] * elements
-    # TO-DO
-    
+def merge(arrA, arrB):
+    merged_arr = []
+    while len(arrA) > 0 and len(arrB) > 0:
+      if (arrA[0] < arrB[0]):
+        merged_arr.append(arrA[0])
+        arrA.pop(0)
+      else:
+        merged_arr.append(arrB[0])
+        arrB.pop(0)
+    while len(arrA) > 0:
+      merged_arr.append(arrA[0])
+      arrA.pop(0)
+    while len(arrB) > 0:
+      merged_arr.append(arrB[0])
+      arrB.pop(0)
     return merged_arr
 
-
-# TO-DO: implement the Merge Sort function below USING RECURSION
-def merge_sort( arr ):
-    # TO-DO
-
+import math 
+def merge_sort(arr):
+  if (len(arr) <= 1):
     return arr
+  else:
+    midpoint = math.floor(len(arr) / 2)
```

Comment:
I can't remember if // does this in every language or if it is just a python thing, but len(arr) // 2 is the same as math.floor(len(arr)/2)